websites/H/HomeAssistant/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+    clientId: "812627990382772224"
+}), browsingStamp = Math.floor(Date.now() / 1000);
+  
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "home_assistant_logo" //banner
+  }; 
+  presenceData.startTimestamp = browsingStamp;
+
+  function homeassistant(nome: string, imagetext: string, details: string) {
+    presenceData.smallImageKey = nome;   // SmallImageKey Function
+    presenceData.smallImageText = imagetext;   // smallImageText Function
+    presenceData.details = details;    // Details Function
+  }
+
+  //#region Main
+  if (document.location.pathname.includes("/lovelace")) {
+    homeassistant("home_assistant_logo,", "Home Assistant", "Home")
+  }
```

Comment:
your key and smallimagetext is always the same... so just set it at the top...

for the others maybe do like Viewing the homepage etc. not just Home